docs/content/en/docs/tasks/write-tasks/_index.md

```diff
@@ -73,5 +73,41 @@ The Lifecycle Toolkit passes the values defined inside the `map` field as a JSON
 At the moment, multi-level maps are not supported.
 The JSON object can be read through the environment variable `DATA` using `Deno.env.get("DATA");`.
 Kubernetes secrets can also be passed to the function using the `secureParameters` field.
-Here, the `secret` value is the K8s secret name that will be mounted into the runtime and made available to the function
-via the environment variable `SECURE_DATA`.
+
+Here, the `secret` value is the name of the K8s secret containing a field with the key `SECURE_DATA`.  
```

Comment:
Are suggestions deliberately disabled here?

I think Kubernetess should be spelled out here rather than K8s